src/reducers/updateColWidth.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule updateColWidth
+ */
+
+'use strict';
+
+/**
+ * Update our cached col width for a specific index
+ * based on the value from colWidthGetter
```

Comment:
This line is false until https://github.com/karry08/fixed-data-table-2/pull/2/ is ported